pages/api/v2/episode/[id].tsx

```diff
@@ -151,10 +151,13 @@ async function fetchCoverImage(id: string, available = false) {
       return [];
     }
 
-    const getData = getProviderWithMostEpisodesAndImage(data);
+    // const getData = getProviderWithMostEpisodesAndImage(data);
     // const getData = data?.[0]?.data;
 
-    return getData.data;
+    const tmdbCover = data.find((object) => { return object.providerId === "tmdb" });
+    
+    if(tmdbCover) return tmdbCover.data;
```

Comment:
so this will force the function to only return tmdb cover? no fallback for other providers?

Reply:
Yes this force to return only tmdb cause tvdb doesnt show correct data for the shows that have more than 1 season and for the fallback there is already consumet's api.